### PR TITLE
[1.13.x-blue] [KIECLOUD-654] - prepare RHPAM and RHPAM Kogito operators for 7.13.1 …

### PR DESCRIPTION
**Pull request by spolti**
…release

Signed-off-by: spolti <fspolti@redhat.com>

Many thanks for submitting your Pull Request :heart:! 

Please make sure your PR meets the following requirements:

- [ ] You have read the [contributors' guide](https://github.com/kiegroup/kogito-operator/blob/main/README.md#contributing-to-the-kogito-operator)
- [ ] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [ ] Pull Request contains a link to the JIRA issue
- [ ] Pull Request contains a description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a unit test that verifies it
- [ ] You've ran `make before-pr` and everything is working accordingly
- [ ] You've tested the new feature/bug fix in an actual OpenShift cluster
- [ ] You've added a [RELEASE_NOTES.md](https://github.com/kiegroup/kogito-operator/blob/main/RELEASE_NOTES.md) entry regarding this change

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>Run operator BDD testing</b>
  Please add comment: <b>/jenkins test</b>

* <b>Run RHPAM operator BDD testing</b>
  Please add comment: <b>/jenkins RHPAM test</b>
</details>

### CODE REVIEW

**Review by ricardozanini — APPROVED** [review]

**Review by radtriste — CHANGES_REQUESTED** [review]
Please revert go.mod and go.sym changes... This makes the build cli failing ...

**Comment by radtriste** [comment]
> Please revert go.mod and go.sym changes... This makes the build cli failing ...

Nevermind. I cannot reproduce locally ...

**Comment by radtriste** [comment]
@spolti looks like the `-mod=mod` build option is the one working each time ...

File: .ci/jenkins/Jenkinsfile [hunk 1]
```diff
@@ -129,7 +129,10 @@ pipeline {
 }
 
 void cleanGoPath() {
-    sh 'rm -rf $GOPATH/bin/*'
+    sh '''
+        go clean -modcache
+        rm -rf $GOPATH/*
+    '''
```
Comment (radtriste) [inline .ci/jenkins/Jenkinsfile:135]: I think that this could be rollbacked

**Pull request merged**